x-pack/plugins/data_visualizer/public/application/index_data_visualizer/utils/saved_search_utils.test.ts

```diff
@@ -107,26 +150,24 @@ describe('getQueryFromSavedSearchObject()', () => {
       version: true,
     });
     expect(getQueryFromSavedSearchObject(kqlSavedSearch)).toEqual({
+      query: { query: 'responsetime > 49', language: 'kuery' },
```

Comment:
Is it worth adding a test for a complex filter, with an `OR` clause for example?

Reply:
Added here [`265d633` (#149459)](https://github.com/elastic/kibana/pull/149459/commits/265d6332a292c0281d2d91754b89d52764e4abeb)